### ISSUE TITLE
Giving multiple player names throws error

### ISSUE THREAD

**Issue by amithm7** [acrisci/playerctl#111]
Using the command, `playerctl --player=spotify,vlc play-pause`
throws the following error

```bash
(playerctl:13815): GLib-GIO-CRITICAL **: g_dbus_connection_signal_subscribe: assertion 'sender == NULL || (g_dbus_is_name (sender) && (connection->flags & G_DBUS_CONNECTION_FLAGS_MESSAGE_BUS_CONNECTION))' failed

(playerctl:13815): GLib-GIO-CRITICAL **: g_dbus_connection_signal_subscribe: assertion 'sender == NULL || (g_dbus_is_name (sender) && (connection->flags & G_DBUS_CONNECTION_FLAGS_MESSAGE_BUS_CONNECTION))' failed

(playerctl:13815): GLib-GIO-CRITICAL **: g_dbus_connection_call_sync_internal: assertion 'bus_name == NULL || g_dbus_is_name (bus_name)' failed
```

**Comment by acrisci** [comment]
I can't reproduce on v2.0.1. What does `playerctl --version` say?

**Comment by amithm7** [comment]
v0.5.0

:( Using Debian stable
I think i have to update it.

**Comment by amithm7** [comment]
Resolved after updating to v2.0.1
